### PR TITLE
Drop init() function wrapping grunt.initConfig()

### PR DESCRIPTION
**Pull request by benjaminapetersen**
I don't think this `init()` function wrapping `grunt.initConfig()` is doing anything.  @jeff-phillips-18 @dtaylor113 curious, is there a best practices reason for having it?

### CODE REVIEW

**Review by jeff-phillips-18 — APPROVED** [review]
I don't see a reason for it.

**Review by dtaylor113 — APPROVED** [review]
LGTM

File: Gruntfile.js [hunk 1]
```diff
@@ -3,8 +3,6 @@ module.exports = function (grunt) {
 
   require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks);
 
-  function init () {
-
     grunt.initConfig({
```
Comment (spadgett) [inline Gruntfile.js]: Should we fix the indentation if we make this change?

Comment (benjaminapetersen) [reply Gruntfile.js]: +1 done

**Review by spadgett — APPROVED** [review]

**Comment by benjaminapetersen** [comment]
New bot is active in this repo?

**Comment by jwforres** [comment]
@benjaminapetersen only partially, there is no test/merge queue for it

**Pull request closed without merging**